issue_template.md

```diff
@@ -24,7 +20,7 @@ ________________________________________________________________________________
 <!--
 ________________________________________________________________________________________________________________________________________
 ________________________________________________________________________________________________________________________________________
-Specify the content phase where this bug belongs to, for example "1-19" or "20-29", etc...
+2) Specify the content phase where this bug belongs to, for example "1-19" or "20-29", etc...
```

Comment:
Hope you don't mind a suggestion for a minor sentence structure improvement.
```suggestion
2) Specify the content phase this bug belongs to, for example "1-19" or "20-29", etc...
```

Reply:
I sure dont mind a helpful hand, thx!

Reply:
Looks like this suggestion got skipped, maybe I should have added a secondary text line underneath it to highlight that there are (at least) 2 ways of phrasing this sentence:
```
2) Specify the content phase this bug belongs to, for example "1-19" or "20-29", etc.
2) Specify the content phase where this bug belongs, for example "1-19" or "20-29", etc.
```